perun-core/src/main/java/cz/metacentrum/perun/core/entry/FacilitiesManagerEntry.java

```diff
@@ -1150,6 +1152,23 @@ public void removeFacilityContact(PerunSession sess, ContactGroup contactGroupTo
 	}
 
 	@Override
+	public List<Service> getAssignedServices(PerunSession sess, Facility facility) throws InternalErrorException, FacilityNotExistsException, PrivilegeException {
+		Utils.checkPerunSession(sess);
+		getFacilitiesManagerBl().checkFacilityExists(sess, facility);
+		
+		// Authorization
+		if (!AuthzResolver.isAuthorized(sess, Role.VOADMIN, facility)
```

Comment:
I don't think, that VOADMIN and VOOBSERVER should be able to use this. For Resource it's ok, since they can somehow manage Resource, but facility settings belong to facility manager.